content/_data/upgrades/2-440-2.adoc

```diff
@@ -0,0 +1,13 @@
+NOTE: These upgrade are not specific to the 2.440.2 release, but are crucial for Jenkins users.
```

Comment:
Kevin wanted to highlight issues that users might encounter at the same time that they were upgrading to the new LTS release.  They could encounter the same issues in other conditions and at other times, but LTS upgrades are a specific event where it seemed helpful to tell users that they need to take specific actions to reduce the risk of issues

Reply:
Due to the two changes mentioned being breaking changes if they were not done/done properly, it felt as though it would be good to call attention to them. However, it's not necessarily enough content that would call for a separate blog post or announcement in that fashion.

Outside of the plugin release notes, the core upgrade guide seemed like the next best place to put this information. I'm totally open to alternative locations for this sort of update to go, as I'd like to make sure things are as clear as possible and without cluttering anything up.